guides/leave_space_mutation.md

```diff
@@ -0,0 +1,50 @@
+---
+copyright: 'Copyright IBM Corp. 2017'
+link: 'leave-space-mutation'
```

Comment:
`leave-space` might be sufficient?

Reply:
Makes sense to me. I've seen both links with and without "mutation" 
@jarusso What do you recommend?